### DISCUSSION TITLE
"Новозаблокированные" аниме сайты стали выдавать "ERR_TIMED_OUT"

### DISCUSSION THREAD

**Discussion by MORFIUS-INC** [bol-van/zapret#917] [Q&A]
Конфиг:

TPWS_OPT="
--hostspell=HoST
--split-pos=1
--nosplice
--disorder
--tlsrec=sni
--oob
--oob-data=0xff
"
![nnm1](https://github.com/user-attachments/assets/943498c9-3f82-497c-b6f3-8383e612425a)

Сам сайт:

https://yummyanime.tv/

**Comment by bol-van** [comment]
nosplice в linux только в 2-3 раза увеличит нагрузку на процессор
эта опция для windows из-за проблем с совместимостью с WSL

**Reply by MORFIUS-INC** [reply]
Отключил, спасибо. У меня все равно отдельный копм на линуксе раздает интернет и фильтрует трафик запретом, по этому не особо запаривался с загрузкой проца.
Я к нему только через рдп подключаюсь. Пингмашина)
